Employee.cs

```diff
@@ -0,0 +1,29 @@
+﻿public class Employee : Person
+{
+    public bool IsEmployeeOfMonth { get; set; }
+    public bool isFired { get; set; }
+
+    public Employee(string firstName, string lastName, int age, bool isEmployeeOfMonth) : base(age, firstName, lastName)
+    {
+        this.IsEmployeeOfMonth = isEmployeeOfMonth;
+        this.isFired = false;
```

Comment:
default value of bool variable is false, so no need to set it in constructor for property isFired